m4/cf3_platforms.m4

```diff
@@ -39,3 +39,4 @@ AM_CONDITIONAL([HPUX], [test -n "`(echo ${target_os} | egrep 'hpux|hp-ux')`"])
 AM_CONDITIONAL([FREEBSD], [test -n "`(echo ${target_os} | grep freebsd)`"])
 AM_CONDITIONAL([NETBSD], [test -n "`(echo ${target_os} | grep netbsd)`"])
 AM_CONDITIONAL([XNU], [test -n "`(echo ${target_os} | grep darwin)`"])
+AM_CONDITIONAL([SHUNIT], [test -n "`which shunit2`"])
```

Comment:
I fixed the shunit2 detection to work on any POSIX platform which is the immediate problem you were running into, but if someone ran these tests on Solaris with shunit2 on their path, they would fail. I made the test script compatible with just about everything, but the package module itself won't work on Solaris without tweaking, and it really doesn't make sense to write that with anything other than FreeBSD in mind.

I'll rewrite the checks to be FreeBSD && shunit, I guess? (Slightly worried no-one will actually run them because of that requirement)